include/ignition/fuel_tools/ClientConfig.hh

```diff
@@ -149,7 +149,7 @@ namespace ignition
 
       /// \brief List of servers the client will connect to.
       /// \return The list of servers.
-      public: std::vector<ServerConfig> Servers() const;
+      public: std::vector<ServerConfig> & Servers() const;
```

Comment:
This breaks ABI: https://build.osrfoundation.org/job/ignition_fuel-tools-abichecker-any_to_any-ubuntu_auto-amd64/263/API_5fABI_20report/

But you can add a new function that returns the reference.